### ISSUE TITLE
Hidden sections without visible modules present can cause confusion.

### ISSUE THREAD

**Issue by lpagitz** [gjbarnard/moodle-format_topcoll#149]
**Describe the bug**
First of all, thanks for your work! My report is related to the latest commit 81e15d8bba3b734ada4106afff4b0128b57e4318 which changed the behaviour of sections in a way that they are not shown when there are no visible modules in them. I am not sure why this change was made and I was wondering if it is related to [this discussion](https://moodle.org/mod/forum/discuss.php?d=452918)?
With this change, when you have let's say 4 sections in a course and sections 1-3 are empty, only section 4 is shown. On the left hand side of the section there is a '4' and on the right hand side it says 'Topic 4'. 
I fear there might be some confusion for users asking where sections 1-3 are and why it starts with 4.

**To Reproduce**
Steps to reproduce the behaviour:
1. Create a new course with 4 sections.
2. Leave sections 1-3 empty, put something (a label) into section 4.
3. Turn off edit mode.
4. Be confused.

**Expected behaviour**
I am not sure how to solve this as the discussion linked in the forum is still ongoing. 

**Screenshots**
All sections visible: ![withedit](https://github.com/gjbarnard/moodle-format_topcoll/assets/26457530/1fab07a0-3328-4fd5-b1b1-66b01e6fbaa1)

Only section 4 visible:
![withoutedit](https://github.com/gjbarnard/moodle-format_topcoll/assets/26457530/ffb911db-aa3e-4862-8e07-e1dce54f583a)

**Versions (please complete the following information):**
 - Moodle: 4.4
 - Format: 81e15d8bba3b734ada4106afff4b0128b57e4318 
 - Browser and version: Chrome 124.0.6367.63 Windows 10 64 bit

**Comment by gjb2048** [comment]
Good point, need to think, not related to https://moodle.org/mod/forum/discuss.php?d=452918 but rather https://moodle.org/mod/forum/discuss.php?d=457863.

**Comment by gjb2048** [comment]
Also take it that the commits (https://github.com/gjbarnard/moodle-format_topcoll/commits/main/) between formal releases (https://github.com/gjbarnard/moodle-format_topcoll/releases) are 'development' where anything can happen / go wrong.

**Comment by lpagitz** [comment]
Thank you, Gareth. I understand and am not using the dev release in production. I will read the discussion you linked to gain more insight on this topic.
